src/content/en/2020/markup.md

```diff
@@ -469,21 +452,20 @@ Standard elements are those that are or were part of the HTML specification. Whi
   <figcaption>{{ figure_link(caption="Low probabilities of finding a given element in pages of the sample.", sheets_gid="184700688", sql_file="pages_element_count_by_device_and_element_type_present.sql") }}</figcaption>
 </figure>
 
-We're including these elements to give an idea what elements may have gone out of favor. But while `dir` and `basefont` were last specified in XHTML 1.0 (2000), the rare use of `rp`, which has been mentioned [as early as 1998](https://www.w3.org/TR/1998/WD-ruby-19981221/#a2-4) but which is also [still part of HTML](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-rp-element), may just suggest that Ruby markup is not very popular.
+We're including these elements to give an idea what elements may have gone out of favor. But while `dir` and `basefont` were last specified in XHTML 1.0 (2000) and are no longer part of HTML, the rare use of `rp` (which was mentioned [as early as 1998](https://www.w3.org/TR/1998/WD-ruby-19981221/#a2-4) and is [still part of HTML](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-rp-element)), may just suggest that Ruby markup is not very popular.
+
 
 ### Custom elements
 
 The 2019 edition of the Web Almanac handled [custom elements](../2019/markup#custom-elements) by discussing several non-standard elements. This year, we found it valuable to have a closer look at custom elements. How did we determine these? Roughly by looking at [their definition](https://html.spec.whatwg.org/multipage/custom-elements.html#custom-elements-core-concepts), notably their use of a hyphen. Let's focus on the top elements, in this case elements used on ≥1% of all URLs in the sample:
 
-{# TODO(authors, analysts): Clarify occurrences and percentages _of what_. Pages? Elements? #}
-
 <figure>
   <table>
     <thead>
       <tr>
         <th>Element</th>
-        <th>Occurrences</th>
-        <th>Percentage</th>
+        <th>Pages</th>
```

Comment:
Off the top of my head, I don’t think this is what the data are about. The occurrences here are *not* pages—they are the number of elements.

We need to be really careful with these labels—a small change in wording can change the whole meaning, as in this case. I strongly advise against just changing them in passing, and I’m also nervous that we (the authors and analysts) get pulled in again for which is going to be a lot more work than it looks like, for given the nature of these edits, we need to review the data sets and should really double-check what is being changed.

So while I get the intent here to make things more clear, ironically it comes with a significant risk of actually making things less clear, or even wrong. I wouldn’t touch the labels anymore.

Reply:
That's the biggest thing I think we need to clarify here! The chapter uses "Occurrences" in one place to mean pages (i.e. multiple occurrences on a page still only count as 1), and in other  places to actually mean "Occurrences" (i.e. multiple occurrences on a page count each time).

Looking at both the data and the query I've attempted to clarify which really are occurrences and which are pages. It's entirely possible I got it wrong, so yes it would be good for @Tiggerito to review, but there is little doubt in my mind that the current chapter is presenting incorrect data.

For example this one has the following in the chapter:

![image](https://user-images.githubusercontent.com/10931297/103458049-4cddb580-4cfc-11eb-91b1-d3bbadceeee5.png)

Which relates to this data [from Sheets](https://docs.google.com/spreadsheets/d/1Ta7amoUeaL4pILhWzH-SCzMX9PsZeb1x_mwrX2C4eY8/edit#gid=184700688) 

![image](https://user-images.githubusercontent.com/10931297/103458064-726abf00-4cfc-11eb-9da1-ca89612bb6d8.png)

Therefore it seems clear that "occurrences" here means pages? But that is not clear to the reader.
As a third check[ the query](https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/sql/2020/03_Markup/pages_element_count_by_device_and_custom_dash_elements.sql) also says this is pages.

Later on you have this:

![image](https://user-images.githubusercontent.com/10931297/103458092-afcf4c80-4cfc-11eb-9ee9-d52f68a314ff.png)

Now we don't have 10million pages so we know "Occurrences" here does not mean pages.

I feel this double usage of the same word to mean two things is NOT clear and you are presenting incorrect data to the readers. And the fact you and I aren't even sure which it is nails that point for me!

So I feel this is something we MUST correct. As I say I've spent considerable time looking at the data and the queries and _think_ I've got them right, but could well be wrong!

Reply:
The first one is from the script:

pages_element_count_by_device_and_element_type_present.sql

The word present is meant to indicate I'm just counting if there are any. Percent of pages they are used on.

The second one comes from:

pages_element_count_by_device_and_element_type_frequency.sql

Where the word frequency is meant to indicate a full count. So that would be occurrences.

Reply:
OK so just to be 100% clear, you agree changing this line (which is about the first example) from “Occurrences” to “Pages”, like I’ve done in this PR, is correct?

The second one (headings table) is further down the page and I just included in this comment to make my point that we’re currently using the same word to mean different things - which is the primary thing I’m trying to correct in this PR (Qing with a few other smaller things). I agree this second “headings” table should stay as “Occurrences” as clearly it’s not pages since there more than the 7.5 million pages we check across mobile and desktop 🙂

Reply:
Correct. I think the first example should say Pages as you did.

Reply:
Hey—first, I think this is _awesome_ to look into improving the chapter. What I’d like to raise is, is what we’re improving here worth the effort (involving time of currently 3 people) and the risk (of something slipping through)?

In my mind this is a clear “no”—we have no indication that something is wrong, AFAIK no one complained about the wording, and generally it seems this is nice, yes, but also rather minor.

I respect other views here and I’m also willing to support this, but I am doubtful and concerned.

A side note, I want to underline the suggestion to give people a heads-up where possible, and I also suggest to break up PRs where possible. The header/label topic would feel much better to me had it been separated from the other changes—now those changes will need to wait, and so does another PR I was prepared to get out (I might get it ready regardless now, accepting a light risk of conflicts as this also needs to get out).

Reply:
As I stated when opening this PR, the motivation for this was not to endlessly tweak or criticise this chapter. During the first translation it was noticed we still had open TODOs in this chapter. This left me with one of four choices"
1. Ignore them and leave open TODOs in each translation - as they are in the Dutch version. If we ever addressed these in the future then we'd have to revisit every translation. I don't think this was a great option to be honest.
2. Assume all the TODOs were in fact done and close them. The chapter is what it is, and we were obviously happy to publish so the TODOs were not that important. That was an option but without looking at the remaining TODOs - which were obviously not addressed sufficiently to warrant removing them.
3. Ask Authors to investigate them. As far as I'm concerned this had been asked once before.
4. Spend considerable time looking through each TODO, closing out the ones that either had been addressed or were not a big deal (e.g. the document size one), and make suggestions to fix the other ones so they could be closed out. This is the choice I made despite it being the most effort on my part. Perhaps in hindsight this was the wrong choice.

> In my mind this is a clear “no”—we have no indication that something is wrong

We have a clear indication something is wrong! Occurrences in one place means number of elements, and in other means number of pages. You then argued that "occurrences here are not pages—they are the number of elements." but both my and @Tiggerito 's analysis shows that is incorrect — it means pages. I'd prefer each chapter to not knowingly continue to publish incorrect information and have put considerable the effort in to correct it, rather than just raising an issue and asking others to do the work.

> A side note, I want to underline the suggestion to give people a heads-up where possible, and I also suggest to break up PRs where possible. The header/label topic would feel much better to me had it been separated from the other changes—now those changes will need to wait, and so does another PR I was prepared to get out (I might get it ready regardless now, accepting a light risk of conflicts as this also needs to get out).

Yes perhaps this was a mistake of mine. I did break up the table reformats (#1587), as didn't feel those required approval, and to keep this PR small. I thought the wording changes were minor enough to include in this and have them reviewed at same time rather than make you review two PRs. I guess I was wrong about this.

I also like the improvements in your PR, despite there being no heads up and no complaints making them necessary, so happy to take them. I wish we'd closed out the one remaining issue here and merged this before you'd opened that though.